src/internal/packager/images/push.go

```diff
@@ -122,7 +125,17 @@ func Push(ctx context.Context, cfg PushConfig) error {
 					return err
 				}
 
-				if err = pushImage(img, offlineNameCRC); err != nil {
+				err = retry.Do(
+					func() error { return pushImage(img, offlineNameCRC) },
+					retry.OnRetry(func(_ uint, err error) {
+						cfg.OCIConcurrency = 1
```

Comment:
Thoughts on changing `ociconcurrency` to a local variable defined at the start of the component level retry? 

This way if it's a problem with the port forward then this will fail both attempts and concurrency will be reset to the user submitted value. If it's a problem with the iops / network it'll be set to one for the subsequent retries.

Reply:
I think thats a good balance, ill implement